qcnv/test/org/qcmg/cnv/TestFiles.java

```diff
@@ -77,12 +77,10 @@ private static void Txt2BAM(File SAM, File BAM) throws IOException{
 		SAMFileHeader header = reader.getFileHeader().clone();
 		header.setSortOrder(SAMFileHeader.SortOrder.coordinate);
 		
-		SAMOrBAMWriterFactory factory = new SAMOrBAMWriterFactory(header, false, BAM, true);
-		SAMFileWriter writer = factory.getWriter();
-		for( SAMRecord record : reader)
-			writer.addAlignment(record);	
-			
-		reader.close();
```

Comment:
why is the reader no longer being closed?

Reply:
thanks, fixed